apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -317,11 +317,16 @@ def ingest(
     EXTERNAL_IDS_ARRAY_NAME = storage_formats[storage_version][
         "EXTERNAL_IDS_ARRAY_NAME"
     ]
-    PARTIAL_WRITE_ARRAY_DIR = (
-        storage_formats[storage_version]["PARTIAL_WRITE_ARRAY_DIR"]
-        + "_"
-        + "".join(random.choices(string.ascii_letters, k=10))
-    )
+    if index_type == "IVF_PQ":
+        PARTIAL_WRITE_ARRAY_DIR = storage_formats[storage_version][
```

Comment:
Why not follow the same dir structure? The random number part is required in order to support parallel or failed ingestions.

Reply:
Ah, so I did not realize it was for parallel ingestions, I thought just failed ingestions. And so I added logic to delete the group before use in C++. I did this because I didn't want Python and C++ to have to communicate about what the name of the dir was. But I'll update to follow the pattern in this follow-up PR: #554